### PR TITLE
add header & footer

### PR DESCRIPTION
**Pull request by bechi**

### CODE REVIEW

**Comment by tanojaja** [comment]
Logo and combo should be aligned in the center

**Comment by bechi** [comment]
done!

File: stats.html [hunk 1]
```diff
@@ -10,24 +11,58 @@
       <script type="text/javascript" src="http://d3js.org/d3.v3.min.js" charset="utf-8"></script>
       <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.7.0/nv.d3.min.js"></script>
       <script type="text/javascript" src="./js/stats.js"></script>
```
Comment (tanojaja) [inline stats.html]: The route is wrong, should be: src="js/foundation..." and src="js/stats.js"

**Comment by tanojaja** [comment]
ACK. Well done!

**Pull request merged**